.appveyor.yml

```diff
@@ -0,0 +1,118 @@
+build: false
+platform:
+  - x64
+#matrix:
+#  fast_finish: true # kills the build at the first failure
+clone_folder: C:\projects\dbal
+
+## Build matrix for lowest and highest possible targets
+environment:
+  dependencies: current
+  matrix:
+  - db: pgsql
+    php_ver_target: 7.0
+  - db: pgsql
+    php_ver_target: 7.1
+  - db: mssql
+    php_ver_target: 7.0
+  - db: mssql
+    php_ver_target: 7.1
+  - db: sqlsrv
+    php_ver_target: 7.0
+  - db: sqlsrv
+    php_ver_target: 7.1
+  - db: sqlite
+    php_ver_target: 7.0
+  - db: sqlite
+    php_ver_target: 7.1
+  - db: mysql
+    php_ver_target: 7.0
+  - db: mysql
+    php_ver_target: 7.1
+  - db: mysqli
+    php_ver_target: 7.0
+  - db: mysqli
+    php_ver_target: 7.1
+  - db: mariadb
+    php_ver_target: 7.0
+  - db: mariadb
+    php_ver_target: 7.1
+
+## Cache composer bits
+#cache:
+#    - '%LOCALAPPDATA%\Composer\files -> composer.lock'
+init:
+  - SET PATH=C:\Program Files\OpenSSL;c:\tools\php;%PATH%
+  - SET COMPOSER_NO_INTERACTION=1
+  - SET PHP=1
+  - SET ANSICON=121x90 (121x90)
+services:
+#  - mssql2014
+#  - mysql
+#  - postgresql94
+  - iis
```

Comment:
Why is a webserver required at all in our test setup? Is it required for SQL Server to run?